pkg/graph/graph.go

```diff
@@ -221,6 +222,10 @@ func buildDependencyGraph(projects []project.Project, environment string, nodeOp
 
 	log.Debug("adding %d Config nodes to graph...", len(configs))
```

Comment:
Nit: Technically this log line might be wrong if the feature flag is enabled as then they wont be added.